core/domain/stats_jobs_continuous.py

```diff
@@ -411,27 +412,72 @@ def entity_classes_to_map_over(cls):
 
     @staticmethod
     def map(item):
```

Comment:
Added a TODO. I agree, this job will probably be painfully slow for explorations with large numbers of answers (like project euler and welcome).